app/web/test/utils.tsx

```diff
@@ -59,7 +61,7 @@ export function createMatchMedia(width: number) {
 }
 
 export function t(
-  translationKey: string,
+  translationKey: Parameters<TFunction>[0],
```

Comment:
Does this help much with autocomplete anyway?

Reply:
No but just managed to fix it!